.github/workflows/ci.yml

```diff
@@ -61,20 +61,26 @@ jobs:
           cargo clippy --all --all-features
 
   test:
-    runs-on: [self-hosted, Linux, large]
-    container:
-      image: mobilecoin/rust-sgx-base:v0.0.28
+    runs-on: [self-hosted, macOS, X64, cargo]
```

Comment:
I'm using my own computer to test since I'm not able to rely on ones that we have. They are seemingly running out of memory randomly, like 55% of the time. Sometimes they will work for days at a time, and then suddenly stop. 🤷

Reply:
So this is a self hosted runner on my work computer

Reply:
Will try to resolve this another way. That said, if another PR came through it would just run using my runner on my work computer(s)